operator/crd-docs/cr/kiali.io_v1alpha1_ossmconsole.yaml

```
@@ -5,18 +5,19 @@ metadata:
   annotations:
     ansible.sdk.operatorframework.io/verbosity: "1"
 spec:
-  version: "default"
+  version: default
 
   deployment:
     imageDigest: ""
     imageName: ""
-    imagePullPolicy: "IfNotPresent"
+    imagePullPolicy: IfNotPresent
     # default: image_pull_secrets is an empty list
     imagePullSecrets: ["image.pull.secret"]
     imageVersion: ""
     namespace: ""
 
   kiali:
+    graph: pf
```

@jshaughn  re: the graph customizations in the plugin... Do you think we need to prepare for a dict of settings here for the graph (in order to support multiple settings in the future)?

Because right now, this means the `kiali.graph` OSSMC CR setting can only ever be a string (so it only supports this one value, and no other graph customization features can be added under the `graph` yaml here).

Do you think we might have additional graph settings that the OSSMC plugin will need?

In short, I wonder if we need to do this - thus leaving open the ability to add additional values in the future should we need them (plus, naming it `impl` makes the meaning of the setting more clear):

```suggestion
    graph:
      impl: "pf"
```

---

I prefer the `.impl`, it opens up the ability to have other customization and also is consistent with the Kiali CR.